core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1070,6 +1075,22 @@ class WorkspaceService(protected val ctx: RawlsRequestContext,
                                                         dataAccess,
                                                         s1
                                 ) { destWorkspaceContext =>
+                                  // TODO: Without limit to make sure all snapshots are fetched
+                                  val snapshotResponse: Future[SnapshotListResponse] = snapshotServiceConstructor(s1)
```

Comment:
Looking for input on what kind of numbers we might expect to see here and ideas for how best to make sure we get all snapshots.  Looping with a large limit until done?  Is there a way to get all without the limit?

Reply:
Some of the WFL-enabled workspaces could have multiple hundreds of references, so we definitely need to handle large numbers. I think the best way to handle it is by paging through N results at a time, until the next page returns 0 results. 100 could be a good page size.